File: DictionaryProject/Commands/CloneHighlightingAttribute.vb
Public Class CloneHighlightingAttribute
    Inherits Attribute

    Public Sub New(ByVal source As Integer, ByVal destination As Integer)
        _SourceLevel = source
        _DestinationLevel = destination
    End Sub

    Private _SourceLevel As Integer
    Public Property SourceLevel() As Integer
        Get
            Return _SourceLevel
        End Get
        Set(ByVal value As Integer)
            _SourceLevel = value
        End Set
    End Property

    Private _DestinationLevel As Integer
    Public Property DestinationLevel() As Integer
        Get
            Return _DestinationLevel
        End Get
        Set(ByVal value As Integer)
            _DestinationLevel = value
        End Set
    End Property
End Class

File: monowordbuilder/wordbuilderbase/Commands/CloneHighlightingAttribute.cs
using System;

namespace Whee.WordBuilder.Model.Commands
{
	public class CloneHighlightingAttribute : Attribute
	{
	
		public CloneHighlightingAttribute(int source, int destination)
		{
			_SourceLevel = source;
			_DestinationLevel = destination;
		}
	
		private int _SourceLevel;
		public int SourceLevel {
			get { return _SourceLevel; }
			set { _SourceLevel = value; }
		}
	
		private int _DestinationLevel;
		public int DestinationLevel {
			get { return _DestinationLevel; }
			set { _DestinationLevel = value; }
		}
	}
}